src/libs/style.js

```diff
@@ -26,7 +26,8 @@ export const elements = {
     }
   },
   sectionHeader: { color: colors.dark(), fontSize: 16, fontWeight: 600, margin: '0.25em 0' },
-  pageContentContainer: { position: 'relative', flexGrow: 1, display: 'flex', flexDirection: 'column' }
+  pageContentContainer: { position: 'relative', flexGrow: 1, display: 'flex', flexDirection: 'column' },
+  contextBarContainer: { flex: 'none', width: 55, backgroundColor: colors.dark(0.15) }
```

Comment:
When I met with Joy she mentioned wanting to make the background light grey, and the context bar transparent:

![image](https://user-images.githubusercontent.com/5368863/124149176-569f6600-da5e-11eb-9d85-51df41f7d634.png)

Not sure if that's something we should do now or later..

Reply:
I made the background light gray in the PR I have up currently. I'll rebase my PR with this and align to this screenshot in there